### PR TITLE
Add second version of loader button

### PR DESCRIPTION
**Pull request by rmccar**
This is to add a second implementation of the loader button. The second version will be a timed version that will only be disabled for 1 second and then will be become enabled again. This is to reduce the double clicking of submit buttons.

### CODE REVIEW

File: src/components/button/button.js [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
 
-  submitButton() {
+  loaderButton() {
     this.button.classList.add('is-loading');
     this.button.setAttribute('disabled', true);
   }
+
+  timerButton() {
+    this.button.setAttribute('disabled', true);
+    setTimeout(
```
Comment (boxadesign) [inline src/components/button/button.js]: This could be updated to use es6 syntax e.g. `setTimeout(() => {`

**Comment by rmccar** [comment]
[Temporarily disable submit button to guard against double clicks](https://trello.com/c/CvzQKCMM/3446-temporarily-disable-submit-button-to-guard-against-double-clicks)

File: src/components/button/button.dom.js [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
   if (buttons.length && form.length) {
     const SubmitButton = (await import('./button')).default;
     buttons.forEach(button => {
-      new SubmitButton(button, form[0]);
+      if (button.classList.contains('js-timer')) {
+        submitType = 'timer';
+      } else if (button.classList.contains('js-loader')) {
+        submitType = 'loader';
+      }
+      console.log(submitType);
```
Comment (boxadesign) [inline src/components/button/button.dom.js]: Remove log line.

Comment (rmccar) [reply src/components/button/button.dom.js]: Missed that one! Its fixed

**Pull request merged**